src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+
+  &__star:last-child {
+    margin-right: 0;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be no spaces inside the parentheses when specifying a pseudo-class. It should be `.stars__star:last-child` without any space before `last-child`.